src/models/lineWithFocusChart.js

```diff
@@ -494,6 +494,12 @@ nv.models.lineWithFocusChart = function() {
             margin.bottom = _.bottom !== undefined ? _.bottom : margin.bottom;
             margin.left   = _.left   !== undefined ? _.left   : margin.left;
         }},
+        margin2: {get: function(){return margin2;}, set: function(_){
```

Comment:
I think this should probably be "focusMargin" to match the "focusHeight" option naming convention...